utils/docker/Dockerfile.centos:7

```diff
@@ -49,7 +49,7 @@ RUN yum install -y python34-PyYAML.x86_64 PyYAML.x86_64
 RUN yum install -y python2-paramiko python34-paramiko
 RUN yum install -y valgrind rsync
 RUN yum install -y ShellCheck
-RUN yum install -y python2-jwt
+RUN yum install -y python-jwt.noarch
```

Comment:
You shouldn't typically need to add the arch of packages for yum unless you are trying to do something off of the happy path like installing i686 packages on an x86_64 platform, etc.  You should never need to add `.noarch`.